client/dashboard/sites/overview/index.tsx

```diff
@@ -47,10 +95,10 @@ function SiteOverview() {
 				/>
 			}
 		>
-			<VStack alignment="stretch" spacing={ 10 }>
-				<Grid columns={ 4 } rows={ 1 } gap={ 6 }>
-					<SitePreviewCard site={ site } />
-					<VStack className="site-overview-cards" spacing={ 6 }>
+			<VStack alignment="stretch" spacing={ isSmallViewport ? 5 : 10 }>
+				<Grid { ...gridLayout } gap={ spacing }>
```

Comment:
If the `<Grid>` is safe for use then we should add it to the root `.eslintrc` to fix the import warning https://github.com/Automattic/wp-calypso/blob/8afca757bdaf3c66e0b21d75d3203362603d15b2/.eslintrc.js#L508